eng/packages/http-client-csharp-mgmt/generator/TestProjects/Local/Mgmt-TypeSpec/src/Generated/MgmtTypeSpecModelFactory.cs

```diff
@@ -65,16 +65,16 @@ public static ManagedServiceIdentity ManagedServiceIdentity(Guid? principalId =
         }
 
         /// <summary> User assigned identity properties. </summary>
-        /// <param name="clientId"> The client ID of the assigned identity. </param>
         /// <param name="principalId"> The principal ID of the assigned identity. </param>
+        /// <param name="clientId"> The client ID of the assigned identity. </param>
         /// <returns> A new <see cref="Models.UserAssignedIdentity"/> instance for mocking. </returns>
-        public static UserAssignedIdentity UserAssignedIdentity(Guid? clientId = default, Guid? principalId = default)
+        public static UserAssignedIdentity UserAssignedIdentity(Guid? principalId = default, Guid? clientId = default)
```

Comment:
Do we know why the order changed?

Reply:
Not sure if this is exactly the reason but when I check the latest version of `typespec-azure-resource-manager`, they define the `principalId` as the first property.
But anyway, in the (near) future, we would not see this class generated because this is one of the models that come from Azure.ResourceManager.